### ISSUE TITLE
remove \pointAndClickOff and similar?

### ISSUE THREAD

**Issue by fedelibre** [MutopiaProject/MutopiaProject#994]
IMO the sources should not force the point and click to be switched off, because:

1. Mutopia build system will take care of it
2. Contributors may work on updating an existing source more easily if point-and-click is enabled

I've started removing all active (not commented) \pointAndClickOff using `git grep`, then I realized there are also many scores using `#(ly:set-option....`.
Before proceeding I'd like to know your thoughts on this.
The branch is [remove-point-and-click-off](https://github.com/fedelibre/MutopiaProject/tree/remove-point-and-click-off).

Thanks

**Comment by glenl** [comment]
Agreed. The preference is to not force it either way; point-and-click will be on by default and a publish command-line will turn it off.

There should be 2 branches and pull requests. One containing those pieces that result in no change to the currently published piece (`\pointAndClickOff` or `#(ly:set-option 'point-and-click #f)`), and those that would suggest a rebuild (optional but probably wanted.)
